src/TensorFlowNET.Core/tf2.cs

```diff
@@ -0,0 +1,49 @@
+/*****************************************************************************
+   Copyright 2018 The TensorFlow.NET Authors. All Rights Reserved.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+******************************************************************************/
+
+using System;
+
+namespace Tensorflow
+{
+    public static class tf2
```

Comment:
What's this `static class tf2` for? Should it be merged into `public partial class tensorflow { }`?

Reply:
actually tf2 is a class that is from tf2.0 and it cant be merged coz it is a different distinct class....